hub/tasks/x_event_source.py

```diff
@@ -0,0 +1,81 @@
+import logging
+import os
+
+from apscheduler.schedulers.asyncio import AsyncIOScheduler
+
+from hub.tasks.near_events import run_agent
+from hub.tasks.twitter_client import get_latest_mentions
+
+logger = logging.getLogger(__name__)
+
+
+RESET_BLOCK_ID_ON_START = True  # reusing setting name for now
+
+# Path to store the last read tweet id
+TWEET_STATUS_FILE = "last_tweet.txt"
+
+scheduler = AsyncIOScheduler()
+
+# future: pull from registry 'integration' item
+EVENT_SOURCE_CONFIG = {"user_id": "1867270324649160704"}
```

Comment:
Please move it unless to .env, put I prefer to have a final implementation to read user_ids from registry before to merge